### PR TITLE
Configure min.insync.replicas for Kafka cluster

### PR DESCRIPTION
**Pull request by pierDipi**
Increase data availability.

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: hack/lib/strimzi.bash [hunk 1]
```diff
@@ -53,6 +53,7 @@ function install_strimzi_cluster {
           transaction.state.log.min.isr: 2
           inter.broker.protocol.version: "3.2"
           auto.create.topics.enable: "false"
+          min.insync.replicas: 2
```
Comment (matzew) [inline hack/lib/strimzi.bash:56]: I think I tried this before, and there were failures...

Comment (pierDipi) [reply hack/lib/strimzi.bash:56]: Ok, I'll check the errors

**Comment by pierDipi** [comment]
Closing for now since failures aren't strictly related to this problem

**Pull request closed without merging**